Controller/RestController.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/*
+ * This file is part of the Massive CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Massive\Bundle\SearchBundle\Controller;
+
+use Massive\Bundle\SearchBundle\Search\SearchManagerInterface;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * REST API for search
+ *
+ * @author Daniel Leech <daniel@dantleech.com>
+ */
+class RestController
```

Comment:
I would not call this one `RestController`, `SearchController` seems like the better choice.